test/test_heuristics.rb

```diff
@@ -91,6 +91,13 @@ def test_es_by_heuristics
     })
   end
 
+  def test_eslintrc_by_heuristics
+    assert_heuristics({
+      "JSON" => all_fixtures("JSON", ".eslintrc"),
```

Comment:
You have to put an asterix before the dot, otherwise it doesn't match anything ;)
In this case, I think you could simply use
```ruby
    "JSON" => all_fixtures("JSON"),
    "YAML" => all_fixtures("YAML"),
```
as the heuristic should work for any JSON or YAML file, irrespective of the extension.